.github/workflows/windows_ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Windows CI
+on:
+  schedule:
+    - cron: "0 12 * * *" # Daily at 12:00 UTC
+  workflow_dispatch: # allows triggering the workflow run manually
+
+env:
+  DISTUTILS_USE_SDK: 1
+  MSSdk: 1
+
+jobs:
+  win-wheels:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [windows-2019]
+        arch: [AMD64]
+        pyver: ['3.8']
```

Comment:
should this be 3.9?

Reply:
Fair enough, done. Also dropped 3.8 from the other .yml file.